libs/core/src/lib/token/token.component.html

```diff
@@ -2,8 +2,10 @@
     #tokenWrapperElement
     class="fd-token"
     role="option"
+    tabindex="0"
```

Comment:
hope the css for focus is taken care ?
Thank you.

Rest of the code looks fine to me and is as expected.

Reply:
Yes, the tokens receive the dotted focus within the token when they receive tab focus. Note that this PR only changes the functionality for tokens that are not part of a tokenizer